### ISSUE TITLE
Bug not allowing users to use the shutdown command completely

### ISSUE THREAD

**Issue by AlexGustafsson** [AlexGustafsson/homebridge-wol#35]
### Backstory
Users have been having issues where a device would not enter the shutdown state.

### Notes
This issue has been created to follow up on the current issue and document the cause and solution.

### Testing @cr3ative's fix

Run the following commands on the computer running homebridge:
```
# Remove homebridge-wol
npm uninstall -g homebridge-wol
# Clone the fix
git clone https://github.com/cr3ative/homebridge-wol.git
# Enter the directory, install dependencies and "install" the plugin
cd homebridge-wol
npm install
npm link
```

To uninstall, run the following commands on the same computer:
```
# "Uninstall" the plugin
npm unlink homebridge-wol
# Remove the directory
rm -r homebridge-wol
```

**Comment by AlexGustafsson** [comment]
Hi, @cr3ative, I’ve seen your fix in #33. Would you be kind to provide the steps to reproduce the issue as well as any other notes you may have on the issue? The same goes for you, @kloshar4o.

**Comment by cr3ative** [comment]
Of course! 

Steps to reproduce the problem on current master:

- Attach some debug to the current state of the target device, as per the pinger function. I just inserted some console logging to the pinger callback along the lines of "Current state: " and then what the function decided to do.
- Create a device with a shut down command attached to it
- Set the shut down grace time to something reasonable like 10 seconds
- Issue said shut down command while watching pinger output
- You'll see that the device never leaves "Shutting down" state, even after 10 seconds, because the logic in master ignores a state switch from "Shutting down" to "Offline".

Due to this, further commands to attempt to wake the device will be ignored, because "Shutting down" is an intermediate state which is ignored in the command processor.

**Comment by pax0707** [comment]
I ran into this testing my issue.
If you shut down your PC "normally" it goes from online to offline.
If you shutdown it via plugin it stays in shutting down.

**Comment by AlexGustafsson** [comment]
Fixed in 3.2.3 courtesy of @cr3ative.
